packages/satML-plugin/satML-plugin.1.30/opam

```diff
@@ -0,0 +1,39 @@
+(******************************************************************************
+* Please, do not modify this file without contacting the maintainers
+* to discuss the reasons of the modification.
+* In particular, dev-repo is not provided because it's private.
+******************************************************************************)
+
+opam-version: "1.2"
+
+maintainer: "alt-ergo@ocamlpro.com"
+authors: "Alt-Ergo developers"
+license: "CeCILL-C"
+homepage: "http://alt-ergo.ocamlpro.com/"
+bug-reports: "https://github.com/OCamlPro/alt-ergo/issues"
+
+build:
+[
+        ["./configure" "-prefix" "%{prefix}%"]
+        [make "satML"]
+]
+
+install:
+[
+        [make "install-satML" "MANDIR=%{man}%/man1"]
+]
+
+remove:
+[
+        ["rm" "%{prefix}%/lib/alt-ergo/plugins/satML-plugin.cmxs"]
+]
+
+depends:
+[
+        "zarith"
+	"camlzip"
+	"ocplib-simplex" {= "0.3"}
+        "alt-ergo" {= "1.30"}
```

Comment:
please fix the formatting here to be more consistent. (we cannot fix it ourselves due to the request at the top of the file). thanks.